woocommerce/wc-admin
# Adding WCPay payment configuration defaults

Fixes part of #6855

Adds defaults for WCPay and other cleanup around it's usage. 

_Also in this PR:_
- Only visible gateways will now be returned from `/onboarding/payments` endpoint

### Detailed test instructions:

-  Checkout this branch and activeate `wc-admin`.
-   Disable the remote payment tester plugin if active and delete the transient woocommerce_admin_remote_payment_methods_specs.
- Set your country to `US` or other approved country.
- Opt out of marketing.
- Ensure that WCPay is _not_ installed yet.
- Visit payment task, and confirm that WCPay is visible, and that it appears like the first screenshot.
- Confirm that WCPay does _not_ appear in either listing below, aside from WCPay top card.
- Click "Get Started," and WCPay should install and present the "Manage" link.
- Confirm that "Manage" link takes you to the WCPay Connect page.
- Go back to payment task screen, and the WCPay card should now appear as the second screenshot.
- For extra credit, delete the WCPay plugin and then clone, install & activate it via [the branch on this PR](https://github.com/Automattic/woocommerce-payments/pull/2014).
- Now when you click "Finish Setup" it should display the correct help text, and the "Connect" button will take you directly to the Stripe setup page.




### Screenshots

![image](https://user-images.githubusercontent.com/444632/120550042-35891e00-c3a9-11eb-9f70-0497596d4787.png)

![image](https://user-images.githubusercontent.com/444632/120550176-5b162780-c3a9-11eb-816b-ebca88b790b0.png)
